core/src/main/scala/kafka/tools/DumpLogSegments.scala

```diff
@@ -257,7 +257,7 @@ object DumpLogSegments {
     private def parseGroupMetadata(groupMetadataKey: GroupMetadataKey, payload: ByteBuffer) = {
       val groupId = groupMetadataKey.key
       val group = GroupMetadataManager.readGroupMessageValue(groupId, payload)
-      val protocolType = group.protocolType
+      val protocolType = group.protocolType.getOrElse("NONE")
```

Comment:
@ijuma It seemed a little more straightforward to use a default value, but let me know if you disagree.